packages/info/src/index.ts

```diff
@@ -4,7 +4,6 @@ import process from 'process';
 import { argv } from './options';
 
 import { AVAILABLE_COMMANDS, AVAILABLE_FORMATS, IGNORE_FLAGS } from './commands';
-import { configReader, fetchConfig, resolveFilePath, getNameFromPath } from './configParser';
```

Comment:
This file is also unused, can be safely removed. Not really sure why it was introduced, @rishabh3112 any ideas?

Reply:
@anshumanv  this file is being used in some test suites. , 

![Screenshot from 2020-03-22 10-59-57](https://user-images.githubusercontent.com/46647141/77243121-5cd44f00-6c2c-11ea-822f-1fd96062af51.png)

Reply:
I think you forogt to rebuild, passing fine for me after removing the file.

Reply:
And no, not used, the fn you showed is in `index.ts` in info, not in `configParser`.

Reply:
Accidently removed some extra code. My bad ;-)